sigs.yaml

```diff
@@ -24,6 +24,33 @@ workinggroups:
     frequency: weekly
     url: meet.google.com/qkt-yacp-wzm
   contact: {}
+- dir: wg-devsecops
```

Comment:
looks like you used a different directory for the files in this PR?! The dirname in here looks good, please use that :)

Reply:
done